src/Services/ItemUpdateService.php

```diff
@@ -49,7 +49,7 @@ class ItemUpdateService
 	/**
 	 * @var Client
 	 */
-	private $client;
+	public $client;
```

Comment:
Never set global variables to public. Use getter instead.

Reply:
done